### DISCUSSION TITLE
Etsy Api v3 Webhooks Not Found

### DISCUSSION THREAD

**Discussion by TarekIbrahim-TeeLaunch** [etsy/open-api#1073] [Q&A]
I'm looking for webhooks to notify my application when and order is created at etsy, but couldn't find any.

Does the current Etsy version supports webhooks?

**Comment by emergingdzns** [comment]
Nope. We all keep hoping it'll come.

**Comment by GorgonFreeman** [comment]
I also came looking for webhooks, shame. In the meantime I guess the best we can do is have a process poll the endpoints and make a webhook out of that.

**Comment by thisisjake** [comment]
Whilst not ideal, I monitor order confirmation emails from Etsy to determine if an order has come through. This triggers my application to check for new orders.

**Reply by FirebirdDP** [reply]
I was thinking of doing the same.  I am doing a check every 5 minutes.

**Comment by FloderCC** [comment]
Hello, is there any update? Thanks in advance

**Comment by turboturtleD** [comment]
Etsy has been pretty clear that they don’t intend to offer this anytime in the foreseeable future.  Not sure why, it seems basic.  We are told we need to poll.

**Comment by barbmich** [comment]
I found this article talking about webhooks, but since i see everyone saying they are not available - I also attempted several queries to the endpoint suggested in the walkthrough - I presume this is yet another example of AI slop: https://rollout.com/integration-guides/etsy/quick-guide-to-implementing-webhooks-in-etsy

**Reply by turboturtleD** [reply]
Yeah, I don’t think that rollout info is legit.  I haven’t even caught a whiff of an indication they have any intentions to implement webhooks.  Need to poll the server if you want updates.
